mvc/views/tag-helpers/intro.rst

```diff
@@ -1,8 +1,281 @@
+
+Most of the built-in Tag Helpers target existing HTML elements and provide server-side attributes for the element. For example, the ``<input>`` element used in many of the views in the *Views/Account* folder contains the ``asp-for`` attribute, which extracts the name of the specified model property into the rendered HTML. The following Razor markup:
+
+.. code-block:: html
+
+   <label asp-for="Email"></label>
+
+Generates the following HTML:
+
+.. code-block:: html
+
+   <label for="Email">Email</label>
+   
+The ``asp-for`` attribute is made available by the ``aspFor`` property in the ``LabelTagHelper``. See :doc:`authoring` for more information.
+
+Managing Tag Helper scope
+-----------------------------
+
+Tag Helpers can use a combination of ``@addTagHelper``, ``@removeTagHelper``, the "!" opt-out character in the view hierarchy to control when Tag Helpers are available to a view.
+
+``@addTagHelper`` makes Tag Helpers available
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you create a new ASP.NET 5 web app named *AuthoringTagHelpers* (with no authentication), the following *Views/_ViewImports.cshtml* file will be added to your project:
+
+.. literalinclude:: authoring/sample/AuthoringTagHelpers/src/AuthoringTagHelpers/Views/_ViewImportsCopy.cshtml
+   :language: html
+   :emphasize-lines: 2
+   :lines: 1-2
+
+The ``@addTagHelper`` directive makes Tag Helpers available to the view. In this case, the view file is *Views/_ViewImports.cshtml*, which by default is included in all views, which means that the Tag Helpers will be available to all views. The code above uses the wildcard syntax ("*") to specify that all Tag Helpers in the specified assembly (*Microsoft.AspNet.Mvc.TagHelpers*) will be available. The first parameter after ``@addTagHelper`` specifies the Tag Helpers to load (we are using "\*" for all Tag Helpers), and the second parameter "Microsoft.AspNet.Mvc.TagHelpers" specifies the assembly containing the Tag Helpers. *Microsoft.AspNet.Mvc.TagHelpers* is the assembly for the built-in ASP.NET 5 Tag Helpers.
```

Comment:
You clarify built-in Tag Helpers here, maybe we should link to this portion in the previous sections when we mention built-in Tag Helpers.